.github/workflows/rust.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and Deploy Site
+
+env:
+  NODE_VERSION: "12.x"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  Deploy:
+    name: "Deploy"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install latest nightly
```

Comment:
nit: please use stable

Reply:
Changed to stable. Didn't realize rust had adopted wasm bindings/compile targets in stable already. Good call out there! :)